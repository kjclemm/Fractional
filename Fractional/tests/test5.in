100 20
79 183 263
264 199 135
280 153 199
155 115 92
267 141 158
69 103 122
299 151 132
283 114 300
259 182 264
135 30 209
234 233 151
142 236 202
96 72 240
167 191 169
43 45 82
202 121 99
205 287 216
164 39 14
289 193 74
221 229 110
44 31 141
188 66 7
38 181 5
4 271 224
207 99 107
235 196 117
196 260 290
178 210 242
97 247 298
25 259 124
215 1 131
55 40 39
203 249 25
292 232 282
162 63 299
182 216 287
147 219 190
191 105 292
115 88 180
261 54 200
254 225 222
176 278 270
290 240 217
300 18 126
118 222 111
12 28 57
253 295 159
53 253 136
91 198 178
83 64 296
296 131 115
80 171 266
185 133 281
36 157 95
124 290 17
172 126 63
49 42 88
199 212 30
113 261 185
151 269 188
121 170 19
127 78 176
72 137 27
51 220 210
74 293 49
120 35 53
2 109 89
174 69 227
130 180 293
183 167 86
209 3 219
105 12 129
197 0 194
278 283 93
177 206 101
16 154 64
37 147 262
128 50 31
10 145 128
236 59 118
117 238 186
190 112 42
109 9 127
173 197 113
24 194 11
216 82 206
101 166 259
119 266 58
132 71 284
125 291 255
9 248 153
222 76 80
297 213 143
146 169 125
86 70 165
31 279 9
0 144 13
100 300 34
244 275 249
23 209 168
151 577 34 307 14 45
9 261 45 468 151 615
38 253 76 544 58 199
58 330 111 268 90 343
164 339 188 194 141 718
184 678 175 685 25 168
118 206 79 244 77 169
137 429 193 407 41 367
-2 550 55 611 141 551
50 423 121 427 167 342
33 114 190 488 13 370
95 244 197 259 113 353
4 372 71 504 4 536
168 554 184 423 126 680
23 42 103 189 98 537
101 279 181 662 53 286
165 583 30 47 127 443
74 501 195 674 22 323
125 418 31 56 87 615
98 221 61 158 179 661
