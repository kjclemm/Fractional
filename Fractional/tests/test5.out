4
39
25
35
1
10
8
22
45
26
7
8
76
13
1
23
0
32
1
7
